### ISSUE TITLE
Citations for individual treebanks

### ISSUE THREAD

**Issue by robvanderg** [UniversalDependencies/docs#772]
Recently, I wanted to evaluate a parser on as many UD datasets as possible, and I started to request the datasets that are hosted without words in them (UD_English-ESL, UD_French-FTB, UD_Hindi_English-HIENCS, UD_Japanese-BCCWJ/, UD_Arabic-NYUAD/, UD_Mbya_Guarani-Dooley). However, for some of them I signed a contract stating that I have to cite their individual papers. I found this unfair compared to the other ~180 treebanks, so I started to collect all citations for them. Would it make sense to include these somehow in the official repos?

The bibs can be found here:
https://github.com/machamp-nlp/machamp/blob/master/docs/cites.tar.gz

Treebanks without a paper have a default bib with a link to the repo and the treebank creators as authors (if we could find them)

ps. This was an exhausting task, and most likely there are some minor mistakes somewhere. Furthermore, we only got one citation per dataset, to make it fit in a table (and this decision was sometimes somewhat arbitrary).

**Comment by ftyers** [comment]
Thanks for doing that! 

For Kazakh there is also:
```bibtex
@inproceedings{tyers_tl2015,
  author = {Tyers, Francis M. and Washington, Jonathan N.},
  title = {Towards a Free/Open-source Universal-dependency Treebank for Kazakh},
  booktitle = {3rd International Conference on Turkic Languages Processing,
  (TurkLang 2015)},
  pages = {276--289},
  year = {2015},
}
```
For Bambara:
```bibtex
@inproceedings{aplonova_2018,
author = {Aplonova, K. and Tyers, F. M.},
title = {Towards a dependency treebank for Bambara},
booktitle = { Proceedings of the 16th Conference on Treebanks and Linguistic Theories},
pages = {138--146},
year = 2018
}
```

**Comment by robvanderg** [comment]
Ah thanks for the additions, for now we've kept the number of citations to 1, simply because our use case was to put them in a table: https://www.aclweb.org/anthology/2021.eacl-demos.22.pdf . Would you say the one you posted is preferred over the other? (then I'll make sure to replace it in the next version of the paper)

Furthermore, if this is to be integrated into the official UD repo's multiple could be included per treebank of course.

**Comment by ftyers** [comment]
In terms of the Kazakh one, they are both as important. I looked at the table, it seems you could just have double lines for those treebanks that have two papers. Essentially the story is that we both worked independently on the Kazakh treebank and then found out about each other and decided to join forces. The current treebank is a joint effort between the two groups.

**Comment by Stormur** [comment]
Hi! Great work! For Latin LLCT, the correct main citation is:

> Cecchini, F. M., Korkiakangas, T. and Passarotti, M. (2020). A New Latin Treebank for Universal Dependencies: Charters between Ancient Latin and Romance Languages. In Proceedings of the Twelfth International Conference on Language Resources and Evaluation (LREC2020), Marseille, France, May. European Language Resources Association (ELRA).

**Comment by jowagner** [comment]
It would also be great if the preferred citation(s) were always in the treebank's readme. Of course, it would be problematic to push that info in there without approval. Maybe the readme template can be updated and treebank contributors contacted to add a section "Citation". Some treebanks use a section "Reference" in this way, others treat "References" like in a research paper.

**Comment by robvanderg** [comment]
Yes, I think it would be good to have 2 types of (groups of) citations in there:
- Citation(s) for the UD treebank
- Citation(s) for the data source

If these would be documented in the README in a standardized way that would make it much simpler. And by default at least the first one can be filled with a standard tex.

**Comment by dan-zeman** [comment]
So I am considering defining a new line in the machine-readable metadata at the end of the treebank README file. To facilitate processing, it would expect one full BibTeX record on one line (i.e., no line breaks and indentation that human readers may prefer; human-readable citations can still occur in the _References_ subsection). It should then be possible to extract and automatically update a table of citations on the UD web.

I'm not sure whether and how we should expand it to accommodate multiple citations per treebank. The easiest approach would be that there could be multiple BibTeX lines without further specification, and if a user/tool is looking for **the citation**, the one that occurs first would be considered the main one.

```
BibTeX: @inproceedings{udpdtc, author = {Marie Mikulová and Barbora Štěpánková and Daniel Zeman and Jan Štěpánek and Milan Straka and Jan Hajič}, title = {Meet {UD_Czech-PDTC}: A Large and Genre-Rich Treebank in {Universal Dependencies}}, booktitle = {Proceedings of the 15th International Conference on Language Resources and Evaluation (LREC 2026)}, publisher = {European Language Resources Association}, address = {Palma, Spain}, year = 2026}
```

Thoughts?

**Comment by ellepannitto** [comment]
Why not allowing a [CITATION.cff](https://citation-file-format.github.io/) file in the repo?

**Comment by dan-zeman** [comment]
> Why not allowing a [CITATION.cff](https://citation-file-format.github.io/) file in the repo?

I was not aware of this format. I don't remember ever seeing such a file in a repository :-)

It looks like they want it to be a de-facto GitHub standard, which would speak for adopting it. On the other hand, I'm not excited about having the metadata scattered in yet another file, and in a format different from BibTeX. But if people like it, we can try it...

**Comment by AngledLuffa** [comment]
naturally there's an XKCD for that: https://xkcd.com/927/

**Comment by ellepannitto** [comment]
Github automatically produces the bibtex for the entry that you choose as main one...

**Comment by dan-zeman** [comment]
> Github automatically produces the bibtex for the entry that you choose as main one...

Yes, I know (now that I've seen the page you linked). What I don't know is whether they also automatically create their file from my BibTeX (because that's typically the format in which I already have my reference).

Now that I think about it, another cool feature would be to generate a big BibTeX file for the entire UD (a smaller brother of ACL's `anthology.bib` :-)), with BibTeX keys for individual treebanks matching their codes in UD (e.g. `ud-it-kiparlaforest`), so that you can easily cite treebanks you work with. I suppose we would have to generate this ourselves, though.

**Comment by robvanderg** [comment]
Any standard way would be better than what there is now, one additional feature people might be looking for is a categorization. I remember from my search that there were at least two common categories with quite different functions: UD-treebank papers, but also original sources (often for converted treebanks, but also just the raw-data providers).

**Comment by ellepannitto** [comment]
> What I don't know is whether they also automatically create their file from my BibTeX (because that's typically the format in which I already have my reference).

I don't think so but that's easily done...

> Now that I think about it, another cool feature would be to generate a big BibTeX file for the entire UD (a smaller brother of ACL's anthology.bib :-)), with BibTeX keys for individual treebanks matching their codes in UD (e.g. ud-it-kiparlaforest), so that you can easily cite treebanks you work with. I suppose we would have to generate this ourselves, though.

This would be very cool and it's also a reference that should probably be automatically added to all READMEs. However, I still like the idea that treebank creators can add their list of references with different tags, because it has happened to me multiple times that I not only want to cite the treebank itself but I'd also need to find the reference paper where the process is described... And most of the times I just resort to searching in google scholar for the authors.

**Comment by dan-zeman** [comment]
> not only want to cite the treebank itself but I'd also need to find the reference paper

For me the priorities are the opposite: I usually want to cite the paper (if it exists). When I proposed the BibTeX line in README, I meant a reference to a paper. If I feel like pointing at the dataset would also be useful, I just add `\footnote{\url{https://github.com/UniversalDependencies/UD_XXX-YYY}}`. When seeing this CITATION.cff for the first time, I leaned to dismiss it because it looked like it is only for `@software` citations; but when I found the documentation, it turned out that a reference to a paper can be added, too.
